src/index.js

```diff
@@ -36,11 +36,10 @@ class LegacyUrls {
 
   /**
    * Generate URL for the schemas of a Taskcluster service.
+   * The schema must have the version in its name i.e. "v1/whatever.json"
```

Comment:
Must?  I'd agree to "usually", but for example I can imagine a service defining some schemas common to events and APIs that are unversioned.  Basically, I think that's just a convention that lib-api follows for the input and output properties.